tests/Unit/QueryBuilderTest.php

```
@@ -0,0 +1,577 @@
+        'content_id'       => ['Resource ID', 'int'],
+    ];
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        // Stub WP functions the class file may reference at include time or
+        // in method bodies.
+        Functions\stubs([
+            '__'                  => static fn(string $text): string => $text,
+            'apply_filters'       => static fn(string $tag, $value) => $value,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tag'. (undefined)

(UnusedFormalParameter)